File: unix_time/unix_time.js
// ATOMIC FUNCTION:date
// TAGS: time
// LANG: node.js
// FUNCTION: date

var start_val=new Date().year.toString();
var expected_end_val="Sun May 31 2020 21:03:18 GMT-0700 (Pacific Daylight Time)";

var end_val=new Date().toString();;

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
	console.log(start_val);
    console.log('FAIL');
}

File: unix_time/unix_time.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class unix_time {
    // ATOMIC_FUNCTION:  unix_time()
    // TAGS: time, string
    // LANG:  java
    // FUNCTION:  unix_time()
    public String unix_time(){
        LocalDateTime now = LocalDateTime.now();
        DateTimeFormatter pattern =
                DateTimeFormatter.ofPattern("yyyy-MM-dd");
        String format = now.format(pattern);

        return format;
    }
    public static void main(String[] args){
        unix_time u = new unix_time();
        String expected_end_val="2020-05-31";

        String end_val= u.unix_time();
        // TEST
        if(end_val.equals(expected_end_val)) {
            System.out.println("PASS");
        } else {

            System.out.println("FAIL");
        }
    }
}
